docs/api-docs/getting-started/rest-api-authentication.md

```diff
@@ -1,260 +1,210 @@
-- If you have created your connection, you’ll want to update your connection parameters:
-	- Where you previously used the BigCommerce store’s secure hostname, you will instead use the `https://api.bigcommerce.com` gateway URL. As an example, requests to `https://store-abc123.mybigcommerce.com/api/v2/orders/123` or `https://my-custom-store-domain.com/api/v2/orders/123` would instead go to `https://api.bigcommerce.com/stores/{store_hash}/v2/orders/123`.
-- With basic auth, you use an authentication HTTP header to authenticate your connection. With OAuth, you’ll want to use the header:
-	- X-Auth-Token header for your access token. For more information see the article [Authentication](/api-docs/getting-started/authentication).
+* Create an API account appropriate to your use case. Keeping in mind the API endpoints your connections use, create either a store API account or an app API account per the preceding instructions. Configure the account with the correct OAuth scopes for your use case. We recommend adhering to industry standard security practices; add only the essential scopes that your application requires. If you're using an app API account, you can always modify the scope later.
+* If you use one of our [client libraries](/tools-resources), consult the library’s documentation for establishing an optimal OAuth configuration.
+* After you create your connection, update your connection parameters as follows:
+  * Use `https://api.bigcommerce.com` as the gateway URL instead of the BigCommerce store’s secure hostname. For example, route requests that formerly went to `https://store-abc123.mybigcommerce.com/api/v2/orders/123` or `https://my-custom-store-domain.com/api/v2/orders/123` to `https://api.bigcommerce.com/stores/{{store_hash}}/v2/orders/123`.
+  * Rewrite your HTTP request headers to use the `X-Auth-Token` header to pass the API account's `access_token` instead of the `Authorization` header. For more information, see [Authentication](/api-docs/getting-started/authentication).
 
-Rate limiting of API requests works differently for OAuth API connections. To become familiar with OAuth limitations, please see the [Rate Limits](/api-docs/getting-started/basics/best-practices#best-practices_rate-limits).
+Rate limiting works differently for OAuth API connections. For details, see the [Rate Limits section](/api-docs/getting-started/best-practices#api-rate-limits) of our API best practices article.
 
 ## OAuth scopes
 
-Scope limits ability to read or write to data. Set the scopes to the minimum level of access needed to accomplish the task at hand.
+**Scope** grants and limits a program's ability to read and write data. Set the scopes to the minimum level of access your implementation needs.
 
-All OAuth scopes except `default` have `read_only` scopes that allow only `GET` and `HEAD` requests.
+All OAuth scopes except `default` provide `read-only` permissions scopes, so that you can limit some accounts to sending `GET` and `HEAD` requests.
```

Comment:
```suggestion
All OAuth scopes except `default` provide `read-only` permissions scopes so that you can limit some accounts to sending `GET` and `HEAD` requests.
```